mlflow/models/evaluation/base.py

```diff
@@ -1136,6 +1141,13 @@ def scatter_plot(eval_df, builtin_metrics, artifacts_dir):
     :param baseline_model: (Optional) A string URI referring to an MLflow model with the pyfunc
                                       flavor. If specified, the candidate ``model`` is compared to
                                       this baseline for model validation purposes.
+    :param pos_label: The positive label used to compute binary classification metrics such as
```

Comment:
shall we add `pos_label` as a config in `evaluator_config` ?

Reply:
In future we might add more arguments used in metric computation, if add them in the evaluate API arguments, the argument list might be too long.

Reply:
@WeichenXu123 

> the argument list might be too long.

What's the issue if the argument list is too long?

Reply:
For better code readability if we can put it in config, and we can add proper prefix for key to avoid config key conflict.
But, this is just my preference.

@dbczumar What's the purpose of "evaluator_config" ?

Reply:
I actually don't have a strong preference for this. Either way looks fine to me.

Reply:
Given `pos_label` is only used for binary classification models, `evaluator_config` might make more sense.

Reply:
Agreed. Evaluator config sounds good!

Reply:
Updated the code to use `evaluator_config` :)